Guide/tailwindcss.markdown

```diff
@@ -92,18 +92,48 @@ We also need a CSS entry point for Tailwind. Create a new file at `tailwind/app.
 }
 ```
 
-### Adding the build step
+### Integrating with `devenv`
 
-We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
+To start the TailwindCSS build process with the IHP dev server, add the following to your flake.nix:
 
-```makefile
-tailwind-dev:
-	tailwindcss -c tailwind/tailwind.config.js -i ./tailwind/app.css -o static/app.css --watch
+```nix
+...
+ihp = {
+    enable = true;
+    projectPath = ./.;
+    packages = with pkgs; [
+        # Native dependencies, e.g. imagemagick
+        (nodePackages.tailwindcss.overrideAttrs
+            (_: {
+                plugins = [
+                    nodePackages."@tailwindcss/aspect-ratio"
+                    nodePackages."@tailwindcss/forms"
+                    nodePackages."@tailwindcss/language-server"
+                    nodePackages."@tailwindcss/line-clamp"
+                    nodePackages."@tailwindcss/typography"
+                ];
+            })
+        )
+        just
```

Comment:
Is the `just` here correct? What is it used for?

Reply:
Oops, that's what I used instead of `make` before. Thanks for spotting this!